testing/simapp/export.go

```diff
@@ -113,8 +121,12 @@ func (app *SimApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []
 
 	// reinitialize all validators
 	err = app.StakingKeeper.IterateValidators(ctx, func(_ int64, val stakingtypes.ValidatorI) (stop bool) {
+		valBz, err := app.StakingKeeper.ValidatorAddressCodec().StringToBytes(val.GetOperator())
```

Comment:
Maybe this is just a little bit simpler?

```suggestion
		valAddr, err := sdk.ValAddressFromBech32(val.GetOperator())
```